cfsdk/src/test/java/io/harness/cfsdk/CfClientTest.java

```diff
@@ -333,6 +333,42 @@ public void shouldGetFlag_FromCacheAlways_WhenNetworkOnline() throws Exception {
         assertEquals(60, cache.getCacheHitCountForEvaluation("testFlag"));
     }
 
+    @Test
```

Comment:
In other tests in this suite, we were always passing a custom cache. This test ensures that the DefaultCache is used and populated correctly